ui/components/component-library/base-avatar/base-avatar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from '../../ui/box/box';
+
+export const BaseAvatar = ({ size }) => {
+  return (
+    <Box>
+      <div
+        className={classnames('base-avatar', `--size-${size}`)}
+        data-testid="base-avatar"
+      ></div>
+    </Box>
```

Comment:
Added this `div` here so that I can grab the test id to test the component. Another alternative could have been 
writing this component without Box since we are not passing the prop directly to Box.

```
      <div
        className={classnames('base-avatar', `--size-${size}`)}
        data-testid="base-avatar"
      ></div>
```